build.gradle

```
@@ -19,6 +19,7 @@ dependencies {
 	implementation('org.springframework.boot:spring-boot-starter-web')
 	implementation group: 'org.postgresql', name: 'postgresql', version: '42.2.5'
```

For unit tests, it is recommended to use an in-memory database such as H2, so the project does not depend on a database implementation.
```suggestion
	implementation group: 'org.postgresql', name: 'postgresql', version: '42.2.5'
	testImplementation 'com.h2database:h2:1.4.200'
```